9c697d609329ca10c7067a418bba1022/update/sys_script_include_0bb58e71971112109790fefce053af28.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_script_include">
+    <sys_script_include action="INSERT_OR_UPDATE">
+        <access>package_private</access>
+        <active>true</active>
+        <api_name>x_snc_plant_mgmt.get_plant_diseases</api_name>
+        <caller_access/>
+        <client_callable>false</client_callable>
+        <description>Once of script to get the plant diseases from an external source. Do not run this more than once otherwise duplicates will exist.</description>
+        <mobile_callable>false</mobile_callable>
+        <name>get_plant_diseases</name>
+        <sandbox_callable>false</sandbox_callable>
+        <script><![CDATA[var get_plant_diseases = Class.create();
+get_plant_diseases.prototype = {
+    initialize: function() {
+    },
+
+	getPlantDiseases: function(lat, lon) {
+        var diseases = new sn_ws.RESTMessageV2('Plant disease import', 'Get Plant Diseases');
```

Comment:
Add a property in the global scope that holds the API key for this platform and call it from this script include and provide it as a query parameter

Reply:
Hi @SapphicFire. I have removed the API key and used a query parameter instead.